frontend/src/components/Pages/FlotillaSite.tsx

```diff
@@ -41,6 +42,10 @@ export function FlotillaSite() {
                                                 path={`${config.FRONTEND_BASE_ROUTE}/mission/:missionId`}
                                                 element={<MissionPage />}
                                             />
+                                            <Route
+                                                path={`${config.FRONTEND_BASE_ROUTE}/historic`}
+                                                element={<HistoricPage />}
+                                            />
```

Comment:
Would it be more clear to name the page "HistoricMissionsPage"? Alternatively, "MissionHistoryPage".

Reply:
Agree, changed it to HistoricMissionPage